source/includes/read-ops/distinct.java

```diff
@@ -0,0 +1,54 @@
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.model.Filters;
+import com.mongodb.reactivestreams.client.*;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+import reactor.core.publisher.Flux;
+
+public class Distinct {
+    public static void main(String[] args) {
+        String uri = "<connection string URI>";
+
+        MongoClientSettings settings = MongoClientSettings.builder()
+                .applyConnectionString(new ConnectionString(uri))
+                .build();
+
+        try (MongoClient mongoClient = MongoClients.create(settings))
+        {
+            MongoDatabase database = mongoClient.getDatabase("sample_restaurants");
+            MongoCollection<Document> collection = database.getCollection("restaurants");
+
+            // start-distinct
+            DistinctPublisher<String> distinctPublisher = collection.distinct("borough", String.class);
+
+            Flux.from(distinctPublisher)
+                    .doOnNext(System.out::println)
+                    .blockLast();
+            // end-distinct
+
+            // start-distinct-query
+            Bson filter = Filters.eq("cuisine", "Italian");
+
+            DistinctPublisher<String> distinctPublisher = collection.distinct("borough", filter, String.class);
+
+            Flux.from(distinctPublisher)
+                    .doOnNext(System.out::println)
+                    .blockLast();
+            // end-distinct-query
+
+            // start-distinct-comment
+            Bson filter = Filters.and(
+                    Filters.eq("borough", "Bronx"),
+                    Filters.eq("cuisine", "Pizza")
+            );
+
+            DistinctPublisher<String> distinctPublisher = collection.distinct("name", filter, String.class).comment("Bronx pizza restaurants");
```

Comment:
Split this onto multiple lines for readability (my suggestion below might not be the right indentation)
```suggestion
            DistinctPublisher<String> distinctPublisher = collection
                  .distinct("name", filter, String.class)
                  .comment("Bronx pizza restaurants");
```